resources/teamcity/developer/publish-to-downloads-keyman-com.ps1

```diff
@@ -43,35 +43,35 @@ $USERPROFILE = $env:USERPROFILE
 # Preparation
 #
 
-if((Test-Path $upload_path) -ne 0) {
-  Remove-Item -Path $upload_path -Recurse
+if((Test-Path ${upload_path}) -ne 0) {
+  Remove-Item -Path ${upload_path} -Recurse
 }
 
-mkdir $upload_path
+mkdir ${upload_path}
 
 #
 # Build Keyman Compiler WINE archive
 #
 
-if((Test-Path ..\release\$kmcomp_zip) -ne 0) {
-  copy ..\release\$kmcomp_zip ..\$upload_path\$kmcomp_zip
+if((Test-Path ..\release\${kmcomp_zip}) -ne 0) {
+  copy ..\release\${kmcomp_zip} ..\${upload_path}\${kmcomp_zip}
 } else {
   cd bin
   if((Test-Path ..\..\common\schemas\keyboard_info\keyboard_info.source.json) -ne 0) {
     # Keyman versions through -16.0
     copy ..\..\common\schemas\keyboard_info\keyboard_info.source.json .
     copy ..\..\common\schemas\keyboard_info\keyboard_info.distribution.json .
-    & "$7Z_HOME\7z.exe" a -bd -bb0 ..\$upload_path\$kmcomp_zip kmcomp.exe kmcmpdll.dll kmcomp.x64.exe kmcmpdll.x64.dll kmconvert.exe keyboard_info.source.json keyboard_info.distribution.json xml\layoutbuilder\*.keyman-touch-layout projects\
+    & "${7Z_HOME}\7z.exe" a -bd -bb0 ..\${upload_path}\${kmcomp_zip} kmcomp.exe kmcmpdll.dll kmcomp.x64.exe kmcmpdll.x64.dll kmconvert.exe keyboard_info.source.json keyboard_info.distribution.json xml\layoutbuilder\*.keyman-touch-layout projects\
```

Comment:
Heads up about merge conflicts with #14162 where we'll use `SEVENZ_HOME` instead of `7Z_HOME`